browser-test/src/applicant/northstar_common_intake_upsell.test.ts

```diff
@@ -199,7 +195,7 @@ test.describe(
           lastName: 'last',
           dobDate: '2021-06-10',
         }
-        await tiDashboard.createClient(client, true)
+        await tiDashboard.createClient(client)
```

Comment:
**question (non-blocking):** Is this just some general test clean-up?  I'm not seeing how it's related to this PR.

Reply:
No, not worth removing it.  Let's keep it in here.